views/authentication/forms/login.hbs

```diff
@@ -75,16 +75,29 @@
 						<input autofocus aria-label="login" type="submit" id="submit-login" class="btn btn-block btn-primary btn-login" value="{{$t "home.header.link.login"}}" data-testid="submit-login" />
 					{{/if}}
 				</div>
+
+				<div class="form-group" style="text-align:center">
+					<a href="" class="submit-pwrecovery">{{$t "login.link.forgotPassword"}}</a>
+				</div>
+
+				{{#if (getConfig "FEATURE_OAUTH_LOGIN_ENABLED")}}
```

Comment:
should also check that a system type iserv exists, before showing the button

Reply:
done. Also: tested. Shows button locally (where iserv system can be found) and does not show button on dev cluster (where iserv system is not available).